src/app/configs/workspaces.cfg

```diff
@@ -0,0 +1,4 @@
+{
+    "default_workspace_name": "Default",
+    "builtin_workspace_files": "Default.mws"
```

Comment:
should there be a string or a list here?

Reply:
yes, list like 
```
    "builtin_workspace_files": "Classic.mws;Music.mws;Advanced audio editing.mws"
```